src/components/Shortcuts/useCustomShortcuts.js

```diff
@@ -0,0 +1,64 @@
+import { useEffect, useState } from 'react'
+import get from 'lodash/get'
+
+import { useClient } from 'cozy-client'
+import logger from 'cozy-logger'
+import { useI18n } from 'cozy-ui/transpiled/react/I18n'
+
+import {
+  mkHomeMagicFolderConn,
+  mkHomeCustomShorcutsConn,
+  mkHomeCustomShorcutsDirConn
+} from 'queries'
+import { formatShortcuts } from './utils'
+
+export const useCustomShortcuts = () => {
+  const client = useClient()
+  const { t } = useI18n()
+  const [shortcutsDirectories, setDirectories] = useState()
+
+  useEffect(() => {
+    const load = async () => {
+      try {
+        const homeMagicFolderConn = mkHomeMagicFolderConn(t)
+        const folder = await client.query(
+          homeMagicFolderConn.query,
+          homeMagicFolderConn
+        )
+
+        if (!folder) throw Error('No root dir')
+
+        const homeShortcutsDirConn = mkHomeCustomShorcutsDirConn(
+          get(folder, 'data[0]._id')
+        )
+        const { data: folders } = await client.query(
+          homeShortcutsDirConn.query,
+          homeShortcutsDirConn
+        )
+
+        if (folders.length === 0) throw Error('No shortcut dirs')
```

Comment:
pourquoi on throw?

Reply:
techniquement c'est pas une erreur en effet, sur le moment ça me semblait le plus simple de skip directement dans le catch

Reply:
Ouais mais attention, on a par exemple Sentry qui va intercepter les Error, ça risque de flooder un peu. Evitons ça ;)

Reply:
Updated

Reply:
not sure here, I removed the throw statements